Util/RepositoryHelper.php

```diff
@@ -79,6 +81,14 @@ public function findPaginated(
             }
         }
 
+        // @todo Esto se podría abstraer en un DefaultSorting que se pueda extender
+        // @todo Faltaría definir la forma de filtrar por campos de entidades relacionadas
+        foreach($sorting as $entityName => $sort) {
```

Comment:
This should be implemented as a filter, just like the default filter. Maybe `SortFilter`?